test/scraper/meck_pomm_landtag_scraper_detail_test.rb

```diff
@@ -53,6 +53,30 @@ def setup
       }, paper)
   end
 
+  test 'extract details 6/4640 without minisitry' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/meck_pomm_landtag_scraper_detail_6_4640.html')))
+    body = html.search("//table[@id = 'parldokresult']")
+    paper = @scraper.extract(body.at_css('.title'))
+
+    assert_equal(
+        {
+            legislative_term: '6',
+        full_reference: '6/4640',
+        reference: '4640',
+        doctype: Paper::DOCTYPE_MINOR_INTERPELLATION,
+        title: 'Verfahren zur Verwendung zusätzlicher Mittel in der Kulturförderung',
+        url: 'http://www.dokumentation.landtag-mv.de/Parldok/dokument/36869/verfahren-zur-verwendung-zus%C3%A4tzlicher-mittel-in-der-kulturf%C3%B6rderung.pdf',
+        published_at: Date.parse('Tue, 17 Nov 2015'),
+        originators: {
+        people: ['Torsten Koplin'],
+        parties: ['DIE LINKE']
+    },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.